tests/ci_build/Dockerfile.cpu

```diff
@@ -29,7 +29,7 @@ RUN conda create -n py35 python=3.5 && \
 RUN \
     pip install pyyaml cpplint pylint astroid sphinx numpy scipy pandas matplotlib sh \
     		recommonmark guzzle_sphinx_theme mock breathe graphviz \
-		pytest scikit-learn wheel kubernetes urllib3 jsonschema boto3 && \
+		pytest scikit-learn wheel python-kubernetes urllib3 jsonschema boto3 && \
```

Comment:
Should use `kubernetes` with pip.

Reply:
Oops, my bad.  Also what's the correct way of install matching cuda with conda?
```
[2020-04-14T09:03:59.184Z] PackagesNotFoundError: The following packages are not available from current channels:

[2020-04-14T09:03:59.184Z] 

[2020-04-14T09:03:59.184Z]   - cudatoolkit=9.0.176
```

Reply:
We can actually remove `cudatoolkit` from `Dockerfile.gpu`, since we don't use cuDF there. CuDF requires CUDA 10.0+, hence the error.

Reply:
Okay, I just ran Docker build on my Linux machine, and finally `test-python-gpu-cuda9.0` build is fixed.

Reply:
Great!  I think cudatoolkit is a dependency of cudf and can be automatically installed?

Reply:
Let's keep it in the Dockerfile, since the official doc (https://rapids.ai/start.html) explicitly includes `cudatoolkit`.

Reply:
Got it.  Thanks for all the help!